Experiments_toolkit.py

```diff
@@ -166,25 +163,37 @@ def preprocess_with_augmentation(DATA, dictionary, pos_key, augmentation_functio
             window = same_window_size(np_sample[:3], size, 10)
             transpose = window.transpose(0, 2, 1)
```

Comment:
why do you have to transpose the data?

Reply:
That's just what I had to do with the augmentation functions I was using

Reply:
should you undo the transpose then before returning?
By the way, this still seems to return a dict?

Reply:
Ya that's a good idea ill have it trasnpose it back before returning. I also have changed it to not return the dict I just havent pushed it yet